src/commands/tag/README.md

```diff
@@ -19,9 +19,24 @@ datadog-ci tag --level job --tags "go.version:`go version`"
   span for the currently running job.
 - `--no-fail` (default: `false`) will prevent the tag command from failing if there are issues submitting the data.
 - `--tags` is an array of key value pairs of the shape `key:value`. This will be the tags added to the pipeline or job span.
-  The resulting dictionary will be merged with whatever is in the `DD_TAGS` environment variable. If a `key` appears both in `--tags` and `DD_TAGS`, whatever value is in `DD_TAGS` will take precedence.
+  The resulting dictionary will be merged with whatever is in the `DD_TAGS` environment variable and in the `--tags-file` argument.
+- `--tags-file` is a file path to a JSON file that contains the tags in the form:
+  ```json
+  {
+    "key1": "value1",
+    "key2": "value2"
```

Comment:
should we mention that all tags here should be strings?

Reply:
or that it should be a flat map?

Reply:
Good idea let me let me clarify that in the readme